test/framework/src/main/java/org/elasticsearch/test/ExternalTestCluster.java

```diff
@@ -84,7 +84,7 @@ public ExternalTestCluster(
             .putList("node.roles", Collections.emptyList())
             .put("node.name", EXTERNAL_CLUSTER_PREFIX + counter.getAndIncrement())
             .put("cluster.name", clusterName)
-            .put(TransportSettings.PORT.getKey(), ESTestCase.getPortRange())
+            .put(TransportSettings.PORT.getKey(), ESTestCase.getPort())
```

Comment:
I don't think this woud work if there are multiple nodes in the cluster. The same likely goes for `InternalTestCluster`.